net-im/signal-desktop-bin/signal-desktop-bin-1.30.0-r1.ebuild

```diff
@@ -40,6 +41,9 @@ src_prepare(){
 	sed -e 's|\("/opt/Signal/signal-desktop"\)|\1 --start-in-tray|g' \
 		-i usr/share/applications/signal-desktop.desktop || die
 	unpack usr/share/doc/signal-desktop/changelog.gz
+	# Fix Bug 706352
+	chrpath opt/Signal/resources/app.asar.unpacked/node_modules/sharp/vendor/lib/libjpeg.so.8.2.2 -r '$ORIGIN:/target/lib' \
```

Comment:
Just put it on the same line here :)
In future you can wrap the line after `||` without additional `\` needed. Ie, 
```
... ||
    die
```